views/agenda.tmpl

```diff
@@ -70,10 +70,10 @@
                     <th>ID</th>
                     <th>Description</th>
                     <th>Bits</th>
-                    <th>IsAbstain</th>
-                    <th>IsNo</th>
                     <th>Count</th>
-                    <th>Progress</th>
+                    {{ if ne (index .Choices 0).Progress 0.0}}
+                        <th>Progress</th> 
```

Comment:
Why not show progress 0?

Reply:
Once a given agenda has been voted in, especially for the old agendas like lnfeatures, sdiffalgorithm and lnsupport the progress value remains to be zero while voting on the said agendas was done long time ago.
Since some data e.g progress is going to available only at certain statuses, we saw it wise with raedah to hide the progress value if its equal to 0.

Reply:
For a vote that is done, I agree progress 0 is confusing, but I'm thinking of agenda items that are pre-voting or just started voting.  I'm OK for now with hiding it like this when progress is 0, but I suspect it there may be states where showing progress 0 does make sense.

Reply:
I could add a check for "defined" agenda state to address that concern.